docs/fastdds/security/access_control_plugin/access_control_plugin.rst

```diff
@@ -146,44 +146,45 @@ This configuration defines the level of protection that the rule applies to the
-|                |                                       |                                          | ``NONE``         |
-|                +---------------------------------------+------------------------------------------+------------------+
-|                | `Liveliness Protection Kind`_         | ``<liveliness_protection_kind>``         | ``SIGN``         |
-|                |                                       |                                          +------------------+
-|                |                                       |                                          | ``ENCRYPT``      |
-|                |                                       |                                          +------------------+
-|                |                                       |                                          | ``NONE``         |
-|                +---------------------------------------+------------------------------------------+------------------+
-|                | `RTPS Protection Kind`_               | ``<rtps_protection_kind>``               | ``SIGN``         |
-|                |                                       |                                          +------------------+
-|                |                                       |                                          | ``ENCRYPT``      |
-|                |                                       |                                          +------------------+
-|                |                                       |                                          | ``NONE``         |
-+----------------+---------------------------------------+------------------------------------------+------------------+
-| Section        | `Topic Access Rules <topic_rules>`_   | ``<topic_access_rules>``                 | ``<topic_rule>`` |
-+----------------+---------------------------------------+------------------------------------------+------------------+
-
++-----------+-------------------------------------------+------------------------------------------+------------------+
+| **Type**  | **Name**                                  | **XML element tag**                      | **Values**       |
++===========+===========================================+==========================================+==================+
+| Element   | `Allow Unauthenticated Participants`_     | ``<allow_unauthenticated_participants>`` | ``false``        |
+|           |                                           |                                          +------------------+
+|           |                                           |                                          | ``true``         |
+|           +-------------------------------------------+------------------------------------------+------------------+
+|           | `Enable Join Access Control`_             | ``<enable_join_access_control>``         | ``SIGN``         |
```

Comment:
The `<enable_join_access_control>` field can be either `true` or `false` instead of `SIGN`, `NONE` nor `ENCRYPT`

Reply:
good catch :+1: